<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddParamDialog</class>
 <widget class="QDialog" name="AddParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="classCodeLabel">
       <property name="text">
        <string>Class Code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="classCodeLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="basisDeltaSpreadLabel">
       <property name="text">
        <string>Basis Delta Spread</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="basisDeltaSpreadLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="basisVolSpreadLabel">
       <property name="text">
        <string>Basis Vol Spread</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="basisVolSpreadLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="basisPriceSpreadLabel">
       <property name="text">
        <string>Basis Price Spread</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="basisPriceSpreadLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="freeRateLabel">
       <property name="text">
        <string>Free Rate</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="freeRateLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="multiplierLabel">
       <property name="text">
        <string>Multiplier</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="multiplierLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="spreadTypeLabel">
       <property name="text">
        <string>Spread Type</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="spreadTypeComboBox"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="volatilityLabel">
       <property name="text">
        <string>Volatility</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="volatilityLineEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="yieldRateLabel">
       <property name="text">
        <string>Yield Rate</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="yieldRateLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddParamDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddParamDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
